docs/docs/user/references/defaults.md

```diff
@@ -1,10 +1,17 @@
 # Defaults
 
-KPOps has a very efficient way of dealing with repeating settings which manifests as [`defaults.yaml`](/resources/pipeline-defaults/defaults).
+KPOps has a very efficient way of dealing with repeating settings which manifests as [`defaults.yaml`](/resources/pipeline-defaults/defaults). This file provides the user with the power to set defaults for any and all [components](/user/references/components), thus omitting the need to repeat the same settings in [`pipeline.yaml`](/user/references/components).
```

Comment:
would it make sense to also link to `/resources/pipeline-components/pipeline` as `pipeline.yaml` reference?